tools/astarte_import/lib/astarte/import.ex

```diff
@@ -246,21 +263,25 @@ defmodule Astarte.Import do
   end
 
   defp xml_event({:startElement, _uri, _l_name, {_prefix, 'value'}, attributes}, _loc, state) do
-    {:ok, reception_timestamp} = fetch_attribute(attributes, 'reception_timestamp')
+    {:ok, reception_timestamp} =
+      get_attribute(attributes, 'reception_timestamp', Time.utc_now())
+
     {:ok, datetime, 0} = DateTime.from_iso8601(reception_timestamp)
 
     %State{state | reception_timestamp: datetime, element_data: []}
   end
 
   defp xml_event({:startElement, _uri, _l_name, {_prefix, 'object'}, attributes}, _loc, state) do
-    {:ok, reception_timestamp} = fetch_attribute(attributes, 'reception_timestamp')
+    {:ok, reception_timestamp} =
+      get_attribute(attributes, 'reception_timestamp', Time.utc_now())
```

Comment:
Why `Time` and not `DateTime`? By the way, a reception timestamp must be set and so `fetch_attribute/2` might be the best option